internal/handlers/sqlite/msg_listdatabases.go

```diff
@@ -73,7 +74,7 @@ func (h *Handler) MsgListDatabases(ctx context.Context, msg *wire.OpMsg) (*wire.
 			continue
 		}
 
-		stats, err := db.Stats(ctx, nil)
+		stats, err := db.Stats(ctx, &backends.DatabaseStatsParams{Refresh: true})
```

Comment:
@chilagrow the issue is about executing `ANALYZE` less often, but after this PR we will execute as often as before :) We should not use `Refresh: true` there, and also probably in `aggregate`

Reply:
That's right. Thanks for this :pray:

Reply:
If we don't call `ANALYZE` on `aggregate`, postgresql returns negative count of document. I suppose that's okay?

```
    --- FAIL: TestAggregateCompatCollStats/CountAndStorageStats (0.09s)
        --- FAIL: TestAggregateCompatCollStats/CountAndStorageStats/TestAggregateCompatCollStats_ArrayDocuments (0.08s)
            aggregate_stats_compat_test.go:131: 
                        Error Trace:    /home/chi/go/src/github.com/chilagrow/FerretDB/integration/aggregate_stats_compat_test.go:131
                        Error:          Not equal: 
                                        expected: int32(4)
                                        actual  : int64(-1)
                        Test:           TestAggregateCompatCollStats/CountAndStorageStats/TestAggregateCompatCollStats_ArrayDocuments
```

Reply:
> postgresql returns negative count of document

It is not PostgreSQL, it is how we use it in tests. Under normal operations, PostgreSQL refreshes stats in the background itself. In our tests, we don't give it a chance to do so. Running `ANALYZE` everywhere makes tests pass – and hurts the real usage. So, in general, we should not call `ANALYZE` ever*†‡. Exceptions include the `validate` command. So let's use it in test.

Reply:
So in our tests, we can use `validate` to update the stats. And we remove `refresh: true` from all other handler correct?

Reply:
From the issue description:
> `dbStats` and `collStats` should call those methods with `Refresh: true` for now (to do less in this issue)

I think changing those commands will take more time with updating tests, so let's not do it right now

Reply:
Yes sure, as description says. We leave `dbStats` and `collStats` with refresh true. I will fix other handler tests by calling `validate` prior. :+1: